src/pseudopeople/configuration/validator.py

```diff
@@ -152,7 +152,8 @@ def _get_default_config_node(
     configuration.
     """
     try:
-        return default_config[key]
+        config_node: LayeredConfigTree = default_config[key]
```

Comment:
The name of this variable (and the method) is confusing if we are actually expecting a `LayeredConfigTree`, since `ConfigNode` is a different type in that package.